sklearn/multiclass.py

```diff
@@ -203,6 +210,48 @@ def fit(self, X, y):
 
         return self
 
+    def partial_fit(self, X, y):
+        """Incrementally fit the underlying estimators on a batch of samples.
+
+        Throws AttributeError if the underlying estimator class does not
+        support incremental updates via `partial_fit`.
+
+        Parameters
+        ----------
+        X : (sparse) array-like, shape = [n_samples, n_features]
+            Data.
+
+        y : (sparse) array-like, shape = [n_samples] or [n_samples, n_classes]
+            Multi-class targets. An indicator matrix turns on multilabel
+            classification.
+
+        Returns
+        -------
+        self
+        """
+        if not hasattr(self.estimator, "partial_fit"):
+            raise AttributeError("Underlying estimator (%s) does not support "
+                                 "incremental updates!"
+                                 % self.estimator.__class__)
+
+        self.label_binarizer_ = LabelBinarizer(sparse_output=True)
```

Comment:
I was going to ask if it is common to have a large number of n_labels such that setting `sparse_output` to be True here is beneficial here.
But it seems like you are densifying below right?
Why not just `sparse_output=False` here and iterate over Y?